vendor/tags/6/bootstrap.html

```diff
@@ -5,19 +5,43 @@
 // which includes the tag. We don't want to get into a recursive loop.
 if (this.Mobify) return;
 
-// Initial load and followup.
-// If src=1, don't load.
-// If src=-1, total failure, set the opt out cookie.
-// Otherwise just load the script.
-function load(src, callback) {
-    if (+src) return ~src || (doc.cookie = mobifypath + '=; path=/');
+    // Set mobify-path=(blank) indicating a critical failure. Since this is
+    // presumably due to a serious failure like a CDN failure, we serve the
+    // desktop experience on the next reload.
+    function setOptOutCookie() {
+        // This workaround addresses an iOS fallthrough issue, where iOS 8's
+        // "preload top hits" option will load a Mobified page in the
+        // background when typing in an address into the URL bar. Safari
+        // seems to cancel the mobify.js download, resulting in us dropping
+        // a failure cookie.
+        //
+        // We get around this by ignoring mobify.js failures in a background
+        // tab.
+        //
+        // Related ticket: https://mobify.atlassian.net/browse/RTM-280
+        //
+        //if (doc.visibilityState && doc.hidden) {
```

Comment:
@donnielrt 

Personally, I get really freaked out by huge blocks of commented out code. Besides confusion, it's also a bit of an eye sore. If it makes the problem worse, I don't think it should be in the file. If it makes things work better, we should have it be opt-out. 

I think if we want to do opt-in there should be a single line comment saying that some devices might have problems and a link to some documentation of the fixed tag + the code snippet to fix it.

But that's me :). Other opinions?